### ISSUE TITLE
Tooltip is not visible when keyboard focus lands on button or info icon.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#184]
**User Experience:**

If Tooltips are not accessible through keyboard for filter controls then users with visual and motion disabilities will not be able to access the information appearing as tooltip.
Note: User credentials should NOT be included in the bug.

**Repro Steps:**

1. Open Azure portal with valid credentials and create a API Management.
2. Click Developer Portal button to navigate to the portal.
3. Wait portal to load in design mode
4. Navigate to the Left nav and select Pages button.
5. Navigate to the various controls of the "Pages" window.
6. Verify the issue. 

**Actual Result:**
Tooltips of info icon, button are not visible when navigating using keyboard.

**Expected Result:**
Tooltips of info icon, button should be visible when navigating using keyboard.

**Comment by msftedad** [comment]
Hi team, Any update on this issue?
